src/libman/Commands/CacheListCommand.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.CommandLineUtils;
+using Microsoft.Web.LibraryManager.Contracts;
+
+namespace Microsoft.Web.LibraryManager.Tools.Commands
+{
+    /// <summary>
+    /// Defines the cache list command.
+    /// Allows the user to display the contents of the libman cache.
+    /// </summary>
+    internal class CacheListCommand : BaseCommand
```

Comment:
We talked about disabling this command. I don't think we should list the cache contents.